types/ember-plausible/services/plausible.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module 'ember-plausible/services/plausible' {
```

Comment:
ember-plausible doesn't export types yet, so I've added some basic ones here for now.